napari/utils/_proxies.py

```diff
@@ -214,7 +214,7 @@ def _in_main_thread() -> bool:
     General implementation of checking if we are in a proper thread.
     If Qt is available and Application is created then assign :py:func:`in_qt_main_thread` to `in_main_thread`.
     If Qt liba are not available then assign :py:func:`in_main_thread_py` to in_main_thread.
-    IF Qt libs are available but there is no Application ti wil emmit warning and return result of in_main_thread_py.
+    IF Qt libs are available but there is no Application that will emit warning and return result of in_main_thread_py.
```

Comment:
```suggestion
    IF Qt libs are available but there is no Application it will emit warning and return result of in_main_thread_py.
```
Typos, but wrong fix.

Reply:
suggesting 
- https://github.com/codespell-project/codespell/pull/3646
meanwhile -- let's just fix it here from suggestion